includes/cli/class-wc-cli-tracker-command.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * WC_CLI_Tracker_Command class file.
+ *
+ * @package WooCommerce\CLI
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Allows access to tracker snapshot for transparency and debugging.
+ *
+ * @since x.x.x
+ * @package WooCommerce
+ */
+class WC_CLI_Tracker_Command {
+
+	/**
+	 * Registers a command for showing WooCommerce Tracker snapshot data.
+	 */
+	public static function register_commands() {
+		WP_CLI::add_command( 'wc tracker-snapshot', array( 'WC_CLI_Tracker_Command', 'show_tracker_snapshot' ) );
```

Comment:
Keen for ideas on the name of the command - ideally it would be easy to type AND tell you exactly what it does.